### ISSUE TITLE
Token Curated Registry for Media Sites

### ISSUE THREAD

**Issue by Legionof7** [district0x/district-proposals#264]
## **Name**: 
Token Curated Registry for Media Sites
This is a terrible name but and I'd love better suggestions :)

## **Purpose**: 
This TCR has two purposes.

1. List sites that use legitimate information/data in writing their stories and opinion pieces.

2.Reveal the general political leaning of news sites. In other words, transparency is the new objectivity. No news site can be free of bias, but if they reveal their bias than readers can take this into account while they read.

The TCR is designed to be very simple and do a few things very well.

## **Description**: 
There are three actors in this district:

1. Lister:Listers are news sites that want to list their site onto the registry. They put a token deposit when they apply and their deposit is refunded if their site is listed. If the site is rejected, their deposit is distributed to the voting tokenholders.

2.Tokenholder:Tokenholders vote using a [Partial Lock Commit/Reveal](https://github.com/ConsenSys/PLCRVoting) method. They can also call challenges against applications and listings.

3.Reader:Readers are people that utilize the TCR to gain information. 

### **Application**
The application that listers send should include the URL to their news site. They should also state their political leaning (very liberal, liberal, neutral, conservative, very conservative). The articles should not be locked behind a paywall so that voters have enough information to make a decision.

### **Incentives**
Listers are incentivized to apply so that they can get more readers. Tokenholders are incentivzed to deny bad applications in order to claim the deposit. They are also incentivized to accept good applications in order to build a good list that listers want to apply to.

### **More Info**
For more info about TCR, challenges, and parameters, please visit [this.](https://medium.com/@ilovebagels/token-curated-registries-1-0-61a232f8dac7)

## **Ethereum Address**:
0x6098266fd3B487492f73f7967b37c98357851161

**Comment by Bradymck** [comment]
Great proposal. Anything that benefits those outside the direct participants is a great TCR

**Comment by Bradymck** [comment]
@Legionof7 I agree, looking at political leaning from right and left lacks dimension. I'm curious, what's your opinion of tools like this: https://mediabiasfactcheck.com/cctv-america/

**Comment by Bradymck** [comment]
You as well @imdying ^

**Comment by Legionof7** [comment]
> @Legionof7 I agree, looking at political leaning from right and left lacks dimension. I'm curious, what's your opinion of tools like this: https://mediabiasfactcheck.com/cctv-america/

I like the Mediabiasfactcheck site though I have seen complaints (that I haven't fully looked into) about it. The issue with these centralized solutions is that even if they are completely fair (which isn't possible), people still won't trust it. It's hard to say that a decentralized solution isn't trusted unless all the tokenholders were strongly biased one way (and they're economically incentivized not too).

**Comment by Bradymck** [comment]
Hey @Legionof7 I'm starting up a new way to curate proposals for our district spotlight blog post. I'm going to be diving into your proposal a bit more on stream. Though I'm doing it in a way that allows the stream viewers to curate the blog post rather than me. Stay tuned. I'll give you news soon.

**Comment by Legionof7** [comment]
Important Note:This is _not_ a community. Some people have come to me and have been confused by this. The purpose of this isn't to create a platform with all the functions that a community should have. Avsa wrote this which I think covers communities pretty well:https://docs.google.com/document/d/10JfZ-6TC48lDqhJh_lGqIBu5KgcuHHYwdd0vo1pD59Y/edit

This should be a tool to allow discussions within communities to function and as a "database of truth".

**Comment by Bradymck** [comment]
Hey there! Just a reminder. We just launched the District Registry. 

If you are still working on this project, be sure to submit it to the registry if you want to see it built as one of the next districts. 

We will be deprecating the old voting app and Github proposal system now that the registry is live. 

If you aren't working on this be sure to stake your DNT into the districts you want to see remain in the Registry. https://registry.district0x.io/

Your staked tokens act as an 'Upvote' within the registry. Similar to any TCR, you will also receive a portion of the challenge rewards if anyone ever challenges the district. So be sure to migrate any tokens you used in the old voting app. 

I will be streaming today now that the registry looks stable and people are beginning to stake DNT. We will dive in, answer all your questions and help get you started. Hope to see you later today! 

Stream: https://www.twitch.tv/district0x

**Comment by Bradymck** [comment]
Hey @Legionof7,
 
I'm not sure if you're aware but we recently launched a new bounty to migrate your proposals to the actual District Registry: https://registry.district0x.io/

We're replacing the old voting app with the registry. Let me know if you need help but I would love to see you migrate this over so you can claim your DNT.

It does take a 10,000 DNT submit to submit your proposal but this gives you an extra 2000 you can stake in the registry beyond the deposit amount.

Hit me up on Telegram or Discord if you need help or have questions.

Telegram: https://t.me/district0x/75217
Discord: https://discord.gg/P9RQejv

PS, please excuse the canned response. I am encouraging everyone here to start migrating so they can claim their 12000 DNT.
